### PR TITLE
Second pass of fixing linting errors

### PR DESCRIPTION
**Pull request by erikh**
There will be a third pass which I hope to complete either later tonight or tomorrow. For now, this is probably too much to review as is, sorry about that.

acronyms: since golint requires 'IP' and 'ID' to be spelled in upper case, I took it upon myself to make some appropriate conversions of the other acronyms we use a lot. Therefore VXLAN and VLAN are now uppercase throughout the source, where before they were just capitalized.

The comments aren't that great and I apologize for that -- please correct me where I made mistakes. I'm sure I got a few things wrong.

### CODE REVIEW

File: gstate/gstate.go [hunk 1]
```diff
@@ -30,48 +30,52 @@ import (
...
 
+// Version constants. Used in managing state variance.
 const (
 	VersionBeta1 = "0.01"
 )
 
-// specifies various parameters to choose the auto allocation values to pick from
-// this allows mostly hands-free allocation of networks, endpoints, attach/detach
-// operations without having to specify these each time an entity gets created
+// AutoParams specifies various parameters to choose the auto allocation values
```
Comment (mapuri) [inline gstate/gstate.go]: Little rewording needed:

```
// AutoParams specifies various parameters for the auto allocation and resource management for 
// networks and endpoints.  This allows for hands-free allocation of resources without having to
// specify these each time these constructs gets created.
```

File: gstate/gstate.go [hunk 2]
```diff
@@ -30,48 +30,52 @@ import (
...
 }
 
-// specifies parameters that decides the deployment choices
+// DeployParams specifies parameters that decides the deployment choices
 type DeployParams struct {
 	DefaultNetType string `json:"defaultNetType"`
 }
 
-// global state of the network plugin
+// Cfg is the global configuration of the network plugin
```
Comment (mapuri) [inline gstate/gstate.go]:
```
Cfg is the configuration of a tenant
```

**Comment by mapuri** [comment]
Thanks Erik. Changes LGTM other than some minor comments.

>  Therefore VXLAN and VLAN are now uppercase throughout the source, where before they were just capitalized.

I wish `golint` had a way to add project specific abbreviations to it's checks, that way we don't need to manually review these in all subsequent commits.

**Comment by erikh** [comment]
Changes made, will merge with your blessing after tests pass.

**Pull request merged**